### PR TITLE
Updated the [Install via docker compose command]

### PR DESCRIPTION
**Pull request by lavakush07**
### ⚠️ &nbsp;&nbsp;Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [✅ ] I have `npm run build` and `npm run serve` locally before submitting this PR
- [✅ ] I have read through the [Contributing](https://devlake.apache.org/community/) Documentation

# Summary

### Does this close any open issues?
https://github.com/apache/incubator-devlake/issues/4290

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

**Comment by abeizn** [comment]
@lavakush07 Thanks for you pr, Docker Compose was introduced in this version (Docker Inc., v2.13.0), so i think docker-compose is right. Or in the doc, when the user use docker v2.13.0+, them can use docker compose.

**Comment by lavakush07** [comment]
Can I please get more clarity on this as it was mentioned in the website  :
image

**Comment by abeizn** [comment]
> Can I please get more clarity on this as it was mentioned in the website : image

Or you should add a sentence, if the version of docker desktop installed by the user is too low to use `docker compose up -d`, please use `docker-compose up -d`

**Comment by lavakush07** [comment]
> > Can I please get more clarity on this as it was mentioned in the website : image
> 
> Or you should add a sentence, if the version of docker desktop installed by the user is too low to use `docker compose up -d`, please use `docker-compose up -d`

Thank you @abeizn , Updated the PR with the required changes.

File: docs/GettingStarted/DockerComposeSetup.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ sidebar_position: 1
 
 1. Download `docker-compose.yml` and `env.example` from [latest release page](https://github.com/apache/incubator-devlake/releases/latest) into a folder.
 2. Rename `env.example` to `.env`. For Mac/Linux users, please run `mv env.example .env` in the terminal. This file contains the environment variables that the Devlake server will use. Additional ones can be found in the compose file(s).
-3. Run `docker-compose up -d` to launch DevLake.
+3. If [Docker v2.13.0+](https://docs.docker.com/get-docker) Run `docker compose up -d` to launch DevLake or else Run `docker-compose up -d`.
```
Comment (Startrekzky) [inline docs/GettingStarted/DockerComposeSetup.md:20]: The wording needs polishing.

**Review by Startrekzky — DISMISSED** [review]

**Review by abeizn — APPROVED** [review]

**Review by abeizn — APPROVED** [review]

File: docs/GettingStarted/DockerComposeSetup.md (hunk 1 above)
Comment (abeizn) [reply docs/GettingStarted/DockerComposeSetup.md:20]: > The wording needs polishing.

merge first, I will update later.

**Pull request merged**